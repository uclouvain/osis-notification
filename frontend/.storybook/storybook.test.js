/*
 */

import initStoryshots from '@storybook/addon-storyshots';

initStoryshots({
  configPath: 'frontend/.storybook'
});
