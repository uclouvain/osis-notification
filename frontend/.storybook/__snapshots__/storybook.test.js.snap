// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Global component No Notification 1`] = `
<ul
  class="nav navbar-nav"
>
  <li
    class="dropdown"
    id="notification-viewer"
  >
    <a
      aria-expanded="false"
      aria-haspopup="true"
      class="dropdown-toggle"
      data-toggle="dropdown"
      role="button"
    >
      <div
        class="bell"
        data-count="0"
      >
        <span
          class="fas fa-bell"
        />
      </div>
    </a>
     
    <ul
      class="dropdown-menu notification-dropdown"
      style="display: none;"
    >
      <li>
        <a
          class="btn disabled"
        >
          
        Mark all as read
      
        </a>
      </li>
       
      <!---->
       
      <!---->
       
      <li
        class="divider"
        role="separator"
      />
        
      <li
        class="progress"
      >
        <div
          aria-valuemax="100"
          aria-valuemin="0"
          aria-valuenow="100"
          class="progress-bar progress-bar-striped active"
          role="progressbar"
          style="width: 100%;"
        >
          <span
            class="sr-only"
          >
            
          Loading
        
          </span>
        </div>
      </li>
    </ul>
  </li>
</ul>
`;

exports[`Storyshots Global component With Errors 1`] = `
<ul
  class="nav navbar-nav"
>
  <li
    class="dropdown"
    id="notification-viewer"
  >
    <a
      aria-expanded="false"
      aria-haspopup="true"
      class="dropdown-toggle"
      data-toggle="dropdown"
      role="button"
    >
      <div
        class="bell"
        data-count="0"
      >
        <span
          class="fas fa-bell"
        />
      </div>
    </a>
     
    <ul
      class="dropdown-menu notification-dropdown"
      style="display: none;"
    >
      <li>
        <a
          class="btn disabled"
        >
          
        Mark all as read
      
        </a>
      </li>
       
      <!---->
       
      <!---->
       
      <li
        class="divider"
        role="separator"
      />
        
      <li
        class="progress"
      >
        <div
          aria-valuemax="100"
          aria-valuemin="0"
          aria-valuenow="100"
          class="progress-bar progress-bar-striped active"
          role="progressbar"
          style="width: 100%;"
        >
          <span
            class="sr-only"
          >
            
          Loading
        
          </span>
        </div>
      </li>
    </ul>
  </li>
</ul>
`;

exports[`Storyshots Global component With Notifications 1`] = `
<ul
  class="nav navbar-nav"
>
  <li
    class="dropdown"
    id="notification-viewer"
  >
    <a
      aria-expanded="false"
      aria-haspopup="true"
      class="dropdown-toggle"
      data-toggle="dropdown"
      role="button"
    >
      <div
        class="bell"
        data-count="0"
      >
        <span
          class="fas fa-bell"
        />
      </div>
    </a>
     
    <ul
      class="dropdown-menu notification-dropdown"
      style="display: none;"
    >
      <li>
        <a
          class="btn disabled"
        >
          
        Mark all as read
      
        </a>
      </li>
       
      <!---->
       
      <!---->
       
      <li
        class="divider"
        role="separator"
      />
        
      <li
        class="progress"
      >
        <div
          aria-valuemax="100"
          aria-valuemin="0"
          aria-valuenow="100"
          class="progress-bar progress-bar-striped active"
          role="progressbar"
          style="width: 100%;"
        >
          <span
            class="sr-only"
          >
            
          Loading
        
          </span>
        </div>
      </li>
    </ul>
  </li>
</ul>
`;
